hpy/devel/include/universal/hpy.h

```diff
@@ -71,5 +82,11 @@ static inline HPy _HPy_New(HPyContext ctx, HPy h_type, void **data) {
     return h;
 }
 
+static inline _HPy_NO_RETURN void
+HPy_FatalError(HPyContext ctx, const char *message) {
+    ctx->ctx_FatalError(ctx, message);
+    abort();
```

Comment:
is there any particular reason why you put the `abort()` here instead of inside `ctx_FatalError`?

Reply:
Yes.  It's needed for gcc to figure out that the call above never returns.  The `abort()` should never be executed.  Note that it seems to be needed even though we say `_HPy_NO_RETURN` above, I think because it's an inlined function and during inlining it forgets this detail.  I'll write a comment in the test.

Reply:
I think it would be good to have a comment also here in the code